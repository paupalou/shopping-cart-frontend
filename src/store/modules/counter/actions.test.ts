import {
  decrementCounterAction,
  incrementCounterAction, incrementCounterFailureAction,
  incrementCounterRequestAction,
  incrementCounterSuccessAction
} from "./actions";

describe('Counter actions', () => {
  it('Increment action', () => {
    expect(incrementCounterAction()).toMatchSnapshot();
  });

  it('Decrement action', () => {
    expect(decrementCounterAction()).toMatchSnapshot();
  });

  it('Increment request action', () => {
    expect(incrementCounterRequestAction()).toMatchSnapshot();
  });

  it('Increment success action', () => {
    expect(incrementCounterSuccessAction()).toMatchSnapshot();
  });

  it('Increment failure action', () => {
    expect(incrementCounterFailureAction()).toMatchSnapshot();
  });
});
