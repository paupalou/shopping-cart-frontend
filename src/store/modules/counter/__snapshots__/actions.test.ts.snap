// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter actions Decrement action 1`] = `
Object {
  "type": "DECREMENT_COUNTER",
}
`;

exports[`Counter actions Increment action 1`] = `
Object {
  "type": "INCREMENT_COUNTER",
}
`;

exports[`Counter actions Increment failure action 1`] = `
Object {
  "type": "INCREMENT_COUNTER_FAILURE",
}
`;

exports[`Counter actions Increment request action 1`] = `
Object {
  "type": "INCREMENT_COUNTER_REQUEST",
}
`;

exports[`Counter actions Increment success action 1`] = `
Object {
  "type": "INCREMENT_COUNTER_SUCCESS",
}
`;
